editoast/src/main.rs

```diff
@@ -66,8 +67,15 @@ pub fn create_server(
         .extras
         .insert("databases".to_string(), databases.into());
 
+    // Setup CORS
+    let cors = CorsOptions::default()
+        .allow_credentials(true)
```

Comment:
why? I don't think it even does anything, as this only does something [when the origins whitelist isn't a wildcard](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#credentialed_requests_and_wildcards)

Reply:
Indeed